static/service-worker-registration.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+if ('serviceWorker' in navigator) {
+  window.addEventListener('load', function() {
```

Comment:
Agreed with P' @PanJ. From Google's Article on "A user's first visit" and "Improving the boilerplate" at https://developers.google.com/web/fundamentals/instant-and-offline/service-worker/registration:

> Spinning up a new service worker thread to download and cache resources in the background can work against your goal of providing the shortest time-to-interactive experience the first time a user visits your site.

> A simple rule of thumb would be to delay registration until after the load event fires on `window`.